Source/TextToSpeechV1/TextToSpeech.swift

```diff
@@ -40,7 +112,7 @@ public class TextToSpeech {
      - parameter password: The password used to authenticate with the service.
      */
     public init(username: String, password: String) {
-        self.credentials = Credentials.basicAuthentication(username: username, password: password)
```

Comment:
:thinking:

Reply:
Swift is smart enough to figure it out based on the context. Because `self.credentials` has already been declared to have a type of `Credentials`, the type system can figure it out. This is pretty idiomatic for Swift--the type prefix is usually only used when necessary.